packages/terra-slide-panel-manager/src/SlidePanelManager.jsx

```diff
@@ -77,6 +77,8 @@ class SlidePanelManager extends React.Component {
       panelSize = disclosureSizeToPanelSize[manager.disclosure.size];
     }
 
+    const panelBehavior = manager.disclosure.typeConfig && manager.disclosure.typeConfig.panelBehavior ? manager.disclosure.typeConfig.panelBehavior : this.props.panelBehavior;
```

Comment:
```suggestion
    const panelBehavior = manager.disclosure.typeConfig?.panelBehavior : panelBehavior;
```

Reply:
updated here e14565d091a8c877b1048d703bdf93f58e3e0dce

Checking just `manager.disclosure.typeConfig` will set panelBehavior to undefined if typeConfig has other key-values without panelBehavior.